hr_salary_rule_reference/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Salary Rule Reference',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Generic Modules/Human Resources',
+    'author': "Savoir-faire Linux,Odoo Community Association (OCA)",
+    'website': 'https://www.savoirfairelinux.com',
```

Comment:
here your brand.

Reply:
As per the OCA guidelines, the website element should look like: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/__manifest__.py